module-apps/apps-common/WindowManager.hpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+namespace app
+{
+    class WindowManager
```

Comment:
That class name suggest a lot and it is doing almost nothing ? Something will be moved here from ApplicationCommon or windowStack ? If not maybe change it to PopupWindowCounter ?